src/AbstractAlgebra.jl

```diff
@@ -300,7 +299,19 @@ include("AliasMacro.jl")
 include("PrintHelper.jl")
 
 # alternative names for some functions from Base
-export is_empty, is_even, is_equal, is_finite, is_inf, is_integer, is_less, is_odd, is_one, is_real, is_subset, is_valid, is_zero
+export is_empty
+export is_equal
+export is_even
+export is_finite
+export is_inf
+export is_integer
+export is_less
+export is_odd
+export is_one
+export is_real
+export is_subset
+export is_valid
+export is_zero
```

Comment:
I previously merged these `export` statements with the list below, but that broke Hecke: because the `@alias` statements ensure that if an alias is exported, also the "original" is exported and vice versa, and apparently Hecke relies on AA exporting (at least) `isone`... We should add an `import Base.isone` to Hecke instead to make it more resilient (but also should move all exports to the *top* export list in this file)